### DISCUSSION TITLE
MBR and GPFs

### DISCUSSION THREAD

**Discussion by bshashikadze** [vdemichev/DiaNN#167] [Q&A]
Dear Vadim,

I know this parts was discussed a lot but I still have little bit of difficulties to understand and I would be very happy if you could help :) So, I would like to build library with Gas-phase fractions (first uploading GPFs and fasta file plus activating DIA-NNs "Fasta digest.." and "Deep learning based ... predictions", and "Generate spectral library") then I would like to use this library to analyse my "real" DIA runs (so I am deactivating all "predictions", "generate spectral library" and etc., just uploading spectral library back which was generated in a previous step with GPFs), what I do not understand very well is what MBR (which is getting activated automatically I think with "Generate spectral library") will actually do in this case? 

Thank you so much in advance,
Bachi

**Comment by bshashikadze** [comment]
....or maybe when it's written like "MBR should not be used for non-quantitative experiments, that is when you only want to create a spectral library, which you would then use on some other dataset", you also mean the type of workflow mentioned - which involves lib generation with GPFs and then using this to analyze "real" DIA runs?
Thank you

**Reply by vdemichev** [reply]
Hi Bachi,

MBR should only be used for quantitative analyses. So when you are only creating a spectral library from GPF data, MBR should be turned off. When analysing your data with GPF-derived library, you also probably don't want to use MBR because the library will be highly project-specific already.

Best,
Vadim

**Reply by bshashikadze** [reply]
Dear Vadim,

OK perfect! thank you very much!

I would ask also for one more clarification, like in main output each header has specification like PG.Q.Value means protein groups q value, when it comes to Q.value and Global.Q.Value columns I guess they are for precursors right? and one more questions:
-in Fragment.Quant.Raw and Fragment.Quant.Corrected- are the fragments sorted by decreasing score or something like this?

Thank you so much for help and for this amazing tool:)
Bachi

**Reply by vdemichev** [reply]
Yes, q-values without PG or GG prefix are for precursors. 
Fragments are sorted by their reference library intensities, you can obtain the fragment IDs by using --report-lib-info.

**Reply by bshashikadze** [reply]
Thank you very much, sorry I just noticed that answer for both questions are actually specified in "Main output reference". thanks for clarifying it again:)
